src/markdown-pages/install-with-kurl/migrating-csi.md

```diff
@@ -73,3 +73,83 @@ kURL does the following when you use the specs above to migrate data from Rook:
+
+```
+$ kubectl get volumes.longhorn.io -A
+```
+
+In many cases, volume health can be attributed to issues with volume replication. Specifically, when multiple replicas are configured for a volume but not all of them have been scheduled yet. To gain more insight, you can review the individual volumes by editing them.
+
+```
+$ kubectl edit volumes.longhorn.io -n longhorn-system <volume name>
+```
+
+_During the migration process, when running on a single node cluster, the system will automatically scale down the number of replicas to 1 in all Longhorn volumes as a measure to ensure the volumes are in a healthy state before beginning the data transfer. This is done to minimize the risk of a migration failure._
+
+#### Check Longhorn Nodes health
+
+To ensure a successful migration, it is crucial to verify that all Longhorn nodes are in a healthy state: able to handle scheduled workloads, and not over-provisioned. You can check the status of the Longhorn nodes by running the following command:
+
+```
+$ kubectl get nodes.longhorn.io -A
+```
+
+If any node is not reported as "Ready" and "Schedulable", you can obtain more information by individually editing the node and checking its "Status" property.
+
+### Known issues
+
+It is important to note that migrating to a different storage provisioner, such as OpenEBS and Rook, from Longhorn can come with its own set of challenges and difficulties. We have found that the Longhorn storage provisioner has caused several operational issues in the past, which is why we have chosen to prioritize other options.
+
+#### Pods stuck in Terminating or Creating state
+
+One of the most common problems that may arise during the migration process is Pods getting stuck in the Terminating or Creating state. This can happen when the Pods are trying to be scaled down or up but are not able to do so due to some underlying issue. In this case, it is recommended to restart the Kubelet service on all nodes. This can be done by opening new sessions to the nodes and running the command below to restart the Kubelet service.
+
+```
+$ sudo systemctl restart kubelet
+```
+
+#### Restore original number of Volume replicas
+
+To ensure a smooth migration process, when executed on a single node cluster, all Longhorn volumes are scaled down to 1 replica. This is done to make it easier to identify any issues that may arise during the migration, as scaling up the number of replicas can potentially mask the underlying problem. Despite the migration not being successful, the volumes will remain at 1 replica in order to identify the root cause of the failure. If necessary you can restore the original number of replicas by running the following command:
+
+```
+$ sudo kurl longhorn rollback-migration-replicas
```

Comment:
does this need sudo?

Reply:
No, it does not.

Reply:
@emosbaugh Hey, please refresh the diff, I have integrated both Rook and OpenEBS into the same documentation.

Reply:
Those were very similar so I though it was better to keep them together.